content/en/docs/instrumentation/java/getting-started.md

```diff
@@ -20,12 +21,20 @@ Ensure that you have the following installed locally:
 
 ## Example Application
 
-The following example uses a basic spring boot application.
+The following example uses a basic
+[Spring Boot](https://spring.io/guides/gs/spring-boot/) application. If you are
+not using Spring Boot, that's fine — this guide will also work with Apache
+Wicket, Play
+[and more](/ecosystem/registry/?component=instrumentation&language=java)
```

Comment:
This should be updated to match what we agreed to in https://github.com/open-telemetry/opentelemetry.io/pull/2653#discussion_r1184192841.